enhancements/cluster-logging/hypershift-audit-logs.md

```diff
@@ -21,174 +21,124 @@ see-also:
 ## Summary
 
 In a self contained cluster, users can forward application, infrastructure and audit logs.
-In a hypershift hosted cluster, API server audit logs are not exposed in the hosted cluster or on the data plane,
-they are generated in Hosted Control Plane (HCP) pods on a management cluster.
+In a hypershift hosted cluster, API server audit logs are not exposed in the hosted cluster or on the data plane.
+They are generated in Hosted Control Plane (HCP) pods on a management cluster and forwarded via HTTP web-hook.
 
 This proposal deploys the existing Cluster Logging Operator (CLO) on management and hosted clusters.
-The CLO supports both the cluster service provider and cluster service consumer personas by
-permitting destinations to be configured on the hosted control plane (for service provider) and
-the data plane (for service consumer) independently.  Each destination may independently filter
-events forwarded to each destination by way of an optional Policy.
+The CLO provides a ClusterLogForwarder resource (CLF) that collects and forwards logs.
+On the hosted cluster, a CLF can be configured to receive audit logs from the management cluster
+and forward them to customer-configured destinations.
 
-This proposal adds a "Hypershift Logging Operator" (HLO) on the management cluster.
-The HLO combines data-plane and control-plane configuration on the control plane,
-so that the CLO on the _management_ cluster can forward API audit logs for both personas.
-The CLO on the _hosted_ cluster continues to forward other logs for the consumer persona as before.
+On the management cluster, this proposal adds a "Hypershift Logging Operator" (HLO).
+The HLO watches _data-plane_ CLF resources, when it sees one configured to receive audit logs it updates a _control_plane_ CLF to do the forwarding.
+The control plane CLF is also used to forward to Splunk and optionally CloudWatch from the control plane directly.
```

Comment:
This doesn't have to be true, is this enhancement written to be generic or specific to managed ocp?

```suggestion
The control plane CLF can be configured to forward to additional supported outputs from the control plane directly.
```

Reply:
This enhancement is written for managed OCP. The CLO features could be used in other settings, but the HLO is intended specifically for the HCP situation. We could generalize it more but I don't know of any desire to release it outside of OCP at the moment.

Reply:
HCP isn't managed only, there are self-managed HCP deployments.  I don't expect what is built here to be only for managed.  It would be in the regular CLO release payload right?  Documented?

Reply:
+1 I'll be more explicit about what is just a "suggestion" or "example" of using existing CLF features for the ROSA HCP case, vs. what's being proposed as new functionality.